web/styles/input_pill.css

```diff
@@ -33,9 +38,12 @@
         }
 
         .pill-image {
-            height: var(--length-input-pill-image);
-            width: var(--length-input-pill-image);
-            min-width: var(--length-input-pill-image);
+            /* Reduce user pill image lengths by 2px to
+               preserve an apparent border around
+               them, and a square aspect-ratio, even under
+               `box-sizing: border-box` in the area. */
+            height: calc(var(--length-input-pill-image) - 2px);
+            width: calc(var(--length-input-pill-image) - 2px);
```

Comment:
Would it make sense to put the `-2px` in the definition of `length-input-pill-image`?

Reply:
It would certainly be cleaner; there's one other use of `--length-input-pill-image`, but it could take the equivalent `--height-input-pill` value. 

Give me a moment to patch with this.

Reply:
Pushed with this fix.